README.md

```diff
@@ -23,7 +23,7 @@ Installation
 Add the following line to the dependencies of your `Cargo.toml`:
 
 ```
-akd = "0.3"
+akd = "0.4"
```

Comment:
Have we bumped the version to 0.4?

Reply:
Yes due to some unsightly contract changes between lib's, it was necessary to get the CI to not complain.